pkg/fake/cloudprovider.go

```diff
@@ -79,6 +80,8 @@ func (c *CloudProvider) Name() string {
 	return "fake"
 }
 
-func (c *CloudProvider) GetSupportedNodeClasses() []schema.GroupVersionKind {
-	return []schema.GroupVersionKind{}
+func (c *CloudProvider) GetSupportedNodeClasses() []status.Object {
+	nodeClasses := make([]status.Object, 1)
+	nodeClasses[0] = status.Object(&providerv1beta1.EC2NodeClass{})
+	return nodeClasses
```

Comment:
```suggestion
	return []status.Object{&providerv1beta1.EC2NodeClass{}}
```

You can collapse all of this